languages/sqlite/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/kr/pty"
+	"golang.org/x/sys/unix"
+
+	"github.com/replit/prybar/utils"
+)
+
+var Instance = &SQLite{}
+
+type SQLite struct {}
+
+// constructConfigFile generates commands to configure the sqlite CLI.
+// It writes them to a temporary file and returns its pathname.
+func constructConfigFile(config *utils.Config) string {
+	f, err := ioutil.TempFile("", "sqlite-config")
+	if err != nil {
+		panic(err)
+	}
+	defer f.Close()
+
+	// main and continuation prompts
+	// TODO: this probably doesn't handle quotation marks properly
+	f.WriteString(fmt.Sprintf(".prompt '%s' '%s'\n", config.Ps1, config.Ps2))
+
+	return f.Name()
+}
+
+func disableEcho(file *os.File) {
```

Comment:
seems strange that you'd have to do this. The pty passed down from pid1 should be setup in a reasonable mode and sqlite3's internal should alter said mode accordingly. If our default pty config doesn't make sense we should change that in pid1 instead.